storage/src/main/java/tech/pegasys/teku/storage/client/CombinedChainDataClient.java

```diff
@@ -485,4 +486,8 @@ private boolean isRecentData(final UInt64 slot) {
         .getEarliestAvailableBlock()
         .thenApply(res -> res.<BeaconBlockSummary>map(b -> b).or(() -> latestFinalized));
   }
+
+  public SafeFuture<Set<SignedBeaconBlock>> getNonCanonicalBlocksAtSlot(final UInt64 slot) {
+    return historicalChainData.getNonCanonicalBlocksBySlot(slot);
+  }
```

Comment:
This probably needs to wind up support both hot and finalized non-canonical blocks. Could be done in a follow up though.  Retrieving hot blocks by slot would require walking the full protoarray which isn't ideal but is all we can do.

And it's probably going to work out easier to include the finalized canonical blocks than it will be to exclude the non-finalised canonical blocks because of the way ProtoArray works. Given the URL for the Api including both canonical and non-canonical would make sense - I was just expecting only returning non-canonical to be easier but it won't be once we include non-finalised blocks.